mteb/evaluation/evaluators/BitextMiningEvaluator.py

```diff
@@ -84,12 +87,12 @@ def _compute_metrics(
 
         scores = {
             "precision": precision_score(
-                labels, predictions, zero_division=0.0, average="weighted"
+                labels, predictions, zero_division=0, average="weighted"
             ),
             "recall": recall_score(
-                labels, predictions, zero_division=0.0, average="weighted"
+                labels, predictions, zero_division=0, average="weighted"
             ),
-            "f1": f1_score(labels, predictions, zero_division=0.0, average="weighted"),
+            "f1": f1_score(labels, predictions, zero_division=0, average="weighted"),
```

Comment:
In classification we have a lot of class imbalance, should we also use weighted metrics?

Reply:
We can add it but I wouldn't change it.

Reply:
Added ap_weighted and f1_weighted